src/test/java/com/google/api/codegen/ProtoDocumentLinkTest.java

```diff
@@ -47,22 +47,23 @@ public void testMatchProtoLink() {
   }
 
   @Test
-  public void testRDocCommentFixer() {
```

Comment:
The `RDocCommentFixer` just calls the `RubyCommentFormatter` now. The logic has not changed; it's just been moved to MVVM.